internal/controller/server_controller.go

```diff
@@ -812,6 +817,31 @@ func (r *ServerReconciler) applyBiosSettings(ctx context.Context, log logr.Logge
 	return nil
 }
 
+func (r *ServerReconciler) handleAnnotionOperations(ctx context.Context, log logr.Logger, server *metalv1alpha1.Server) (bool, error) {
+	annotations := server.GetAnnotations()
+	operation, ok := annotations[metalv1alpha1.OperationAnnotation]
```

Comment:
Can we do this in one line?

Reply:
we could, but I am using the annotations variable in line 822/837/838

so I would need to call server.GetAnnotations() instead of the variable.